analytics/pyproject.toml

```diff
@@ -10,22 +10,23 @@ version = "0.1.0"
 analytics = "analytics.cli:app"
 
 [tool.poetry.dependencies]
-dynaconf = "^3.2.4"
-kaleido = "0.2.1"
+dynaconf = "^3.2.4" 
+kaleido = "0.2.1" 
 notebook = "^7.0.0" # Goal is to replace this with another method of presenting charts
-pandas = "^2.0.3"
-pandas-stubs = "^2.0.2.230605"
-plotly = "^5.15.0"
-pydantic = "^2.0.3"
-python = "^3.11"
-slack-sdk = "^3.23.0"
-typer = {extras = ["all"], version = "^0.9.0"}
+pandas = "^2.0.3" 
```

Comment:
do we have a linter that catches additional whitespace, and is it configured to run on our .toml files too?

Reply:
Huh, I do have pre-commit setup, which normally runs and flags this -- not totally sure why it didn't catch on this commit though.

Reply:
I fixed this in this commit using `pre-commit` I might have uninstalled it temporarily: [fix: Run pre-commit on pyproject.toml](https://github.com/HHS/simpler-grants-gov/pull/913/commits/825d54a2c428307aa92c7b17e58b2e6a3ddf9bbb)